### ISSUE TITLE
[Bug]: Cloud Storage search crash

### ISSUE THREAD

**Issue by Imactuallymute** [FTBTeam/FTB-Modpack-Issues#944]
### Modpack

Plexiglass Mountain

### Modpack version

1.0.2

### Log Files

[crash-2022-06-12_00.04.15-server.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/8884925/crash-2022-06-12_00.04.15-server.txt)
[crash-2022-06-12_00.07.02-server.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/8884926/crash-2022-06-12_00.07.02-server.txt)

### Describe the bug

While Playing on a server I uploaded a large amount of items to the cloud storage mod's cloud chest and later tried to search up an item within the chest. I got a few letters into searching before the server crashed and kicked me and everyone out. As soon as the server went back up i tried again and had the same result. My guess is that the mod cant handle a lot of varied items under so many names, or it cant handle the volume as the GUI was bugging a bit when scrolling up or down.

### Steps to reproduce

1. Fill Cloud Chest up with items using roughly 20 chests. I have no idea how many items specifically, it was about 4 Diamond crates worth from the mod Better Crates
2. Open the chest attempt to use the search function. Each time i searched up "card" and "car" before it crashed as i was looking for my Mekanism cardboard boxes
3. the crash should only let you get a few letters off

### Expected behaviour

I expected the search function to show off the items matching the title I input into the search bar

### Additional information

I was on a server each time the crash happened. There was a lot of different junk in the cloud storage, imagine everything a casual player would have before starting a big mod. There were containers, items with data or multiple tags, and thousands of one item.

**Comment by Raidobw2** [comment]
Thanks, I've reported it to the mod author above

**Comment by Raidobw2** [comment]
This is not fixed yet, there has not been an update for Cloud Storage for 1.18.2... Cloud Storage will be removed in PGM 1.3.0.

Before removing it, I'm soon releasing 1.2.0 which is a content update.

1.3.0 will be small and serve to better transition the removal of Cloud Storage

This is on hold until 1.3.0

**Comment by slumtrimpet** [comment]
@Raidobw2 

Cloud storage was updated a few days ago and doesn't seem to be abandoned.  Any idea if this will stick around in future releases now?  (will there be future releases of Plexiglass Mountain?)

**Comment by slumtrimpet** [comment]
@Raidobw2 

Cloud storage was updated a few days ago and doesn't seem to be abandoned.  Any idea if this will stick around in future releases now?  (will there be future releases of Plexiglass Mountain?)

**Comment by Raidobw2** [comment]
Hmm, it looks like the 1.18.2 version was not updated, so there are still bugs in that version, which is what Plexi is using. I could look into making an update for Plexi in the summer, but it will depend on my tasks (and I'm part time), so short term this will not be looked into for now.

I can pass the message along the team though. If you say the latest version is actively being worked on, in that case maybe we can include it in future packs. Thanks for the idea!
